templates/configmap.yaml

```diff
@@ -45,7 +45,7 @@ data:
       spi_acs:
         dispatcher_mnt_point:
         data_server_cache:
-        dummy_cache: dummy_prods
+        dummy_cache: /data/dummy_prods
```

Comment:
Why is this necessary? Where was the previous location?

Reply:
just to align it with the the other integral instrument.
otherwise it would be in `/data/dispatcher_scratch/dummy_prods/`

Reply:
does it make sense to you?